QUEST.tsv

```diff
@@ -1499,193 +1499,193 @@ xQUEST_20150323_001498	Ok, pay only 1,000 silver then.
 QUEST_20150323_001499	Good! That's a deal!
 QUEST_20150323_001500	Then don't. Pft.
 QUEST_20150323_001501	You're a bum.
-QUEST_20150323_001502	Here you go. You're even more pathetic than me…
+xQUEST_20150323_001502	Here. You're more pathetic than I am…
 QUEST_20150323_001503	Ah, what's up? 
 QUEST_20150323_001504	Farming alone keeps me busy.
-QUEST_20150323_001505	I'm not sure yet… About that… Rebellion.
-QUEST_20150323_001506	It's enticing but…
-QUEST_20150323_001507	Do you have a written letter of Horacius? 
+xQUEST_20150323_001505	I'm not sure yet… about that… rebellion.
```

Comment:
Old might be better. Person might be talking about the historic Ruklys Rebellion